jetty-jmh/src/main/java/org/eclipse/jetty/http/jmh/MultiPartBenchmark.java

```diff
@@ -277,6 +295,191 @@ public static void main(String[] args) throws RunnerException
         
         new Runner(opt).run();
     }
+
+
+    public static class MultipartExpectations
```

Comment:
This seems unrelated to the PR.

Reply:
Ah, i get it now.
You moved this from jetty-http to jetty-jmh to break the dependency that javadoc freaks out about.